x/fungible/keeper/v2_deposits.go

```diff
@@ -25,25 +25,27 @@ func (k Keeper) ProcessV2Deposit(
 	amount *big.Int,
 	message []byte,
 	coinType coin.CoinType,
+	isCrossChainCall bool,
 ) (*evmtypes.MsgEthereumTxResponse, bool, error) {
 	context := systemcontract.ZContext{
-		Origin:  from,
-		Sender:  ethcommon.Address{},
+		Origin:  []byte{},
```

Comment:
Would it be a big impact to move `from` address from `Origin` to `Sender` (and could be truncated bytes for non-EVM address)?

Reply:
Didn't think about it since we didn't integrate other network with the gateway.

But I think we should have kept a single sender field that is bytes, too late now, I think when integrating Bitcoin, we should use the Origin field

Reply:
Updated the related issues